lib/charms/layer/basic.py

```diff
@@ -278,6 +284,29 @@ def _load_wheelhouse_versions():
     return versions
 
 
+def _add_back_versions(pkgs, _versions):
```

Comment:
I think it would help to describe (e.g. with a docstring) that `_versions` is supposed to be a dictionary where the keys are elements of `pkgs`

Reply:
I'm a bit confused; the docstring is immediately below?